python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -594,12 +594,13 @@ def makeGraph(self, pipeline: Pipeline, args: SimpleNamespace) -> Optional[Quant
         else:
             if _LOG.isEnabledFor(logging.INFO):
                 qg_task_table = self._generateTaskTable(qgraph)
+                qg_task_table_formatted = "\n".join(qg_task_table.pformat_all())
```

Comment:
Are you sure there isn't an astropy Table printing option for print all?

Reply:
`pformat` is an astropy formatting option (https://docs.astropy.org/en/stable/table/access_table.html#pformat-method). Are you thinking of `pprint`? That method prints the table to stdout immediately.